johnsonandjohnson/Bodiless-JS
# feat(gatsby-theme-bodiless): Merge Production Changes on Pull

## Changes

- Check for master updates before "Pull Changes" is executed and merge master first if no conflicts or app rebuild required.

## Test Instructions

#### Test non-conflicting changes:
- Make changes to upstream branch through GitHub UI.
- Make non-conflicting changes to your fork master through GitHub UI.
- Click "Pull Changes"
- Make sure you see both changes locally.
- Make sure these changes also reflected in GitHub ( pushed to GitHub )

#### Test conflicting changes:
- Make changes to upstream branch through GitHub UI.
- Make changes that would cause a conflict to your fork master through GitHub UI.
- Click "Pull Changes"
- Make sure changes from the upstream branch are merged and there is a notification that production changes can not be merged.

## Related Issues
Closes: https://github.com/johnsonandjohnson/Bodiless-JS/issues/304
